### PR TITLE
Fix/Appointment-Status-Update

### PR DESCRIPTION
**Pull request by MonishaSivanathan**
- changed the appointment status with the patient_id and appointment_id
- if no record matched it will raise exception

### CODE REVIEW

File: controller/appointment_controller.py [hunk 1]
```diff
@@ -753,30 +753,45 @@ def custom_query_with_pagination(query_name: str, patient_name, start_date, end_
...
             logger.error(traceback.format_exc())
             error_response_data = {
-                "error": "Unable to retrieve Lab Result",
+                "error": "Unable to retrieve Appointment",
                 "details": str(e),
             }
             return JSONResponse(
                 content=error_response_data,
-                status_code=status.HTTP_400_BAD_REQUEST
+                status_code=status.HTTP_500_INTERNAL_SERVER_ERROR
```
Comment (tamsud) [inline controller/appointment_controller.py:796]: why are you raising the internal server error

File: routes/appointment_routes.py [hunk 2]
```diff
@@ -44,8 +44,8 @@ async def get_appointment(
     return AppointmentClient.get_appointment(patient_name, start_date, end_date, service_name, page, page_size)
 
 
-@router.put("/appointment/{appointment_id}")
-async def update_lab_status(appointment_id: str, update_status: StatusModel, request: Request):
+@router.put("/appointment/{patient_id}")
```
Comment (tamsud) [inline routes/appointment_routes.py:47]: as per my knowledge patient id will bein the payload, why its moved its moved to url?

also router has only patient ID

**Review by tamsud — CHANGES_REQUESTED** [review]

File: controller/appointment_controller.py (hunk 1 above)
Comment (MonishaSivanathan) [reply controller/appointment_controller.py]: changed the status code @tamsud

File: routes/appointment_routes.py (hunk 2 above)
Comment (MonishaSivanathan) [reply routes/appointment_routes.py]: i have changed

**Review by tamsud — APPROVED** [review]

**Pull request merged**